pkgs/tools/virtualization/kubevirt/default.nix

```
@@ -0,0 +1,44 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, testers
+, kubevirt
+}:
+
+buildGoModule rec {
+  pname = "kubevirt";
+  version = "0.52.0";
+  strictDeps = true;
+
+  src = fetchFromGitHub {
+    owner = "kubevirt";
+    repo = "kubevirt";
+    rev = "v${version}";
+    sha256 = "0ldc13nypbk3r9ihbdfibpyvzyzz4m68zzb7g6cq551ks9qdiyma";
+  };
+
+  vendorSha256 = null;
+
+  subPackages = [ "cmd/virtctl" ];
+
+  tags = [ "selinux" ];
+
+  ldflags = [
+    "-X kubevirt.io/client-go/version.gitCommit=v${version}"
+    "-X kubevirt.io/client-go/version.gitTreeState=clean"
+    "-X kubevirt.io/client-go/version.gitVersion=v${version}"
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    package = kubevirt;
+    command = "virtctl version --client";
+    version = "v${version}";
+  };
+
+  meta = with lib; {
+    description = "Client tool to use advanced features such as console access";
+    homepage = "https://kubevirt.io/";
+    license = licenses.apsl20;
```

```suggestion
    license = licenses.asl20;
```

The license is [Apache License 2.0](https://github.com/kubevirt/kubevirt/blob/main/LICENSE) not "Apple Public Source License 2.0".

---

Oops.

---

Fixed